plugins/wacom-usb/fu-wac-module.c

```diff
@@ -96,7 +96,7 @@ fu_wac_module_refresh_cb(FuDevice *device, gpointer user_data, GError **error)
 			    "refresh returned status 0x%x [%s]",
 			    priv->status,
 			    fu_wac_module_status_to_string(priv->status));
-		return FALSE;
+		return priv->status != FU_WAC_MODULE_STATUS_BUSY;
```

Comment:
nah, if we set an error we have to return FALSE. can you check for `priv->status != FU_WAC_MODULE_STATUS_BUSY` before setting the GError and return TRUE there?

Reply:
I think I got my wires crossed when rebasing at some point and put this in the wrong location. It should be up in `fu_wac_module_refresh_cb` to prevent retries in hopeless scenarios. By the time we're in `fu_wac_module_refresh_cb` the retries have already all finished...